### PR TITLE
adds salted oats and peppercorn

### PR DESCRIPTION
**Pull request by Crackaduck**
## About the PR 
Adds in new mutations, salted oats and pepper corn! they contain salt and pepper, and can be grinded into bags of salt and bags of pepper respectively, both of which at the moment have no use. In the future, salt bags and pepper could be used for more recipies, and it gives botany the ability to grow salt and pepper!

## Why's this needed? 
I thought it would be a fun little addition to add, and would also give botany some new things to play with.

## Changelog

```changelog
(u)Carsontheking
(*)Salted oats and Pepper corn added!
(+)Adds in a new crop mutation for oats, salted oats, which contain salt. Also adds in a new crop mutation for corn, pepper corn, which contains pepper. They can also be processed into bags of salt, and bags of pepper respectively, both of which which serve no purpose currently.
```

### CODE REVIEW

**Comment by Crackaduck** [comment]
not sure exactly whats happening with the checks here, its probably because im using a branch i already have a PR on, may have to make a new branch for this

**Comment by Crackaduck** [comment]
anyone got any idea whats happening with the checks?

File: code/modules/food_and_drink/plants.dm [hunk 1]
```diff
@@ -188,6 +188,17 @@
 	plant_reagent = "ethanol"
 	brew_result = "ethanol"
 
+/obj/item/reagent_containers/food/snacks/plant/corn/pepper
+	name = "Pepper corn cob"
+	desc = "Wha? Whys this called corn? Its pepper!."
```
Comment (flrsh) [inline code/modules/food_and_drink/plants.dm:193]:
```suggestion
	desc = "Wha? Why's this called corn? It's pepper!"
```

File: code/modules/hydroponics/plant_mutations.dm [hunk 2]
```diff
@@ -236,6 +242,15 @@
 	assoc_reagents = list("iron")
 	crop = /obj/item/plant/wheat/metal
 
+// Oat Mutations
+
+/datum/plantmutation/oat/saltoat
+	name = "Salted Oat's"
```
Comment (flrsh) [inline code/modules/hydroponics/plant_mutations.dm:248]:
```suggestion
	name = "Salted Oats"
```

File: code/modules/hydroponics/plant_mutations.dm [hunk 3]
```diff
@@ -236,6 +242,15 @@
 	assoc_reagents = list("iron")
 	crop = /obj/item/plant/wheat/metal
 
+// Oat Mutations
+
+/datum/plantmutation/oat/saltoat
```
Comment (flrsh) [inline code/modules/hydroponics/plant_mutations.dm:247]:
```suggestion
/datum/plantmutation/oat/salt
```
To keep things consistent.

File: code/modules/hydroponics/plants_crop.dm [hunk 4]
```diff
@@ -52,6 +52,7 @@ ABSTRACT_TYPE(/datum/plant/crop)
 	endurance = 0
 	genome = 10
 	commuts = list(/datum/plant_gene_strain/growth_fast,/datum/plant_gene_strain/health_poor)
+	mutations = list(/datum/plantmutation/oat/saltoat)
```
Comment (flrsh) [inline code/modules/hydroponics/plants_crop.dm:55]:
```suggestion
	mutations = list(/datum/plantmutation/oat/salt)
```

File: code/modules/food_and_drink/ingredients.dm [hunk 5]
```diff
@@ -263,6 +263,22 @@
 	food_color = "#CC9966"
 	custom_food = 1
 
+/obj/item/reagent_containers/food/snacks/ingredient/salt
+	name = "salt"
+	desc = "A must have in any kitchen, just dont use too much."
```
Comment (flrsh) [inline code/modules/food_and_drink/ingredients.dm:268]:
```suggestion
	desc = "A must have in any kitchen, just don't use too much."
```

File: code/modules/food_and_drink/ingredients.dm [hunk 6]
```diff
@@ -263,6 +263,22 @@
...
+	icon_state = "salt"
+	amount = 1
+	food_color = "#a7927d"
+	custom_food = 1
+
+/obj/item/reagent_containers/food/snacks/ingredient/pepper
+	name = "pepper"
+	desc = "A must have in any kitchen, just dont use too much."
```
Comment (flrsh) [inline code/modules/food_and_drink/ingredients.dm:276]:
```suggestion
	desc = "A must have in any kitchen, just don't use too much."
```

File: code/obj/item/plants.dm [hunk 7]
```diff
@@ -213,6 +213,11 @@
 	desc = "A bland but healthy cereal crop. Good source of fiber."
 	icon_state = "oat"
 
+/obj/item/plant/oat/salt
+	name = "oat"
+	desc = "A salty but healthy cereal crop. Just dont eat too much without water."
```
Comment (flrsh) [inline code/obj/item/plants.dm:218]:
```suggestion
	desc = "A salty but healthy cereal crop. Just don't eat too much without water."
```

File: code/obj/item/plants.dm [hunk 8]
```diff
@@ -213,6 +213,11 @@
 	desc = "A bland but healthy cereal crop. Good source of fiber."
 	icon_state = "oat"
 
+/obj/item/plant/oat/salt
+	name = "oat"
```
Comment (flrsh) [inline code/obj/item/plants.dm:217]:
```suggestion
	name = "salted oat"
```

**Review by flrsh — CHANGES_REQUESTED** [review]
Cool mutations! My requested changes are just minor grammatical things.

File: code/modules/food_and_drink/plants.dm (hunk 1 above)
Comment (Crackaduck) [reply code/modules/food_and_drink/plants.dm]: thanks

File: code/modules/hydroponics/plant_mutations.dm (hunk 2 above)
Comment (Crackaduck) [reply code/modules/hydroponics/plant_mutations.dm]: done, thank you

File: code/modules/hydroponics/plant_mutations.dm (hunk 3 above)
Comment (Crackaduck) [reply code/modules/hydroponics/plant_mutations.dm]: good point, its in

File: code/modules/food_and_drink/ingredients.dm (hunk 5 above)
Comment (Crackaduck) [reply code/modules/food_and_drink/ingredients.dm]: jeez another one huh?

File: code/modules/food_and_drink/ingredients.dm (hunk 6 above)
Comment (Crackaduck) [reply code/modules/food_and_drink/ingredients.dm]: again

File: code/obj/item/plants.dm (hunk 7 above)
Comment (Crackaduck) [reply code/obj/item/plants.dm]: truth be told, i do most of my coding at night, when im tired

File: code/obj/item/plants.dm (hunk 8 above)
Comment (Crackaduck) [reply code/obj/item/plants.dm]: thanks you, for pointing all that out! have a nice day.

File: code/modules/hydroponics/plants_crop.dm (hunk 4 above)
Comment (Crackaduck) [reply code/modules/hydroponics/plants_crop.dm]: sorry if im spamming your mailbox, idk how it works when I comment on them.

**Review by pali6 — APPROVED** [review]

**Comment by pali6** [comment]
I don't see any other issues with grammar so I'm gonna assume @flrsh's approval and merge.

**Pull request merged**